src/features/page/home/dialogs/CompareSimulationsDialog.tsx

```diff
@@ -24,19 +24,20 @@ export default function CompareSimulationsDialog({
   open,
   onClose,
 }: DialogProps): JSX.Element {
-  const { data: mapData } = useGetAllMapsBasicInfoQuery();
+  const { data: mapData, isError: connectionError } = useGetAllMapsBasicInfoQuery();
   const { data: simulationData } = useGetAllSimulationsQuery();
```

Comment:
a tutaj juz nie potrzebujemy errora?

Reply:
no zakladam tak mini ze jak pierwszy nie zwroci error to drugi tez nie, w sensie connection error jak wystapi w drugim totakze w peirwszym wiec bez znaczenai ktore sprawdze (a tylko ten error tu sprawdzam)

Reply:
no moglbys ew zrobic variabla totalError = connectionError1 ?? connectionError2 w sensie tego zalozenia nie widac za bardzo w kodzie, imo tak by bylo czytelniej

Reply:
done